lib/appsignal/utils.rb

```diff
@@ -52,5 +52,15 @@ def encode_utf8(value)
 
       extend ClassMethods
     end
+
+    class Gzip
+      module ClassMethods
+        def compress(body)
+          Zlib::Deflate.deflate(body, Zlib::BEST_SPEED)
+        end
+      end
```

Comment:
Yeah that could work here. I'm just trying to set up a more common structure for modules like this. In this case the module is so small that it's not really necessary to use a ClassMethods module and extend. Only when it gets bigger